### PR TITLE
[Bug] Update Team form labels

### LINKED ISSUE #5868: 🐛 Team Page ( create, edit) has the label as `organization` instead of `team`
## 🐛 Bug

Create Team and Edit Team have the labels as `organization` instead of `team`

## 🦋 Expected Behaviour

Any label consists of `organization` word should be changed to `team`
## 📋 Steps to Reproduce

1.  Navigate to admin/teams/create page and observe  the labels `Organization's Name` for English and French 
2. Navigate to admin/teams/{id} page and observe the labels `Organization's Name` and `Organization's Short Description` 

## 📸 Screenshot
Create team
Screen Shot 2023-03-07 at 4 24 29 PM
Edit team 
Screen Shot 2023-03-07 at 4 24 17 PM
## 🙋‍♀️ Proposed Solution

(optional) Do you have a proposed solution?

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] `Organization's Name`  should be changed to `Team's Name` in English and should be translated accordingly for  French as well 
- [ ] `Organization's Short Description`  should be changed to `Team's Short description` in English and should be translated accordingly for French as well

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #5868 

## 👋 Introduction

Replaces "Organization" for "Team" in the team create/edit forms.

## 📋 TODO

- [x] Add translations

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Build `npm run build`
2. Confirm the team edit/create labels use "Team" instead of "Organization"

## 📸 Screenshot
Screenshot 2023-03-13 084817

### CODE REVIEW

**Comment by vd1992** [comment]
and the view team page too

https://github.com/GCTC-NTGC/gc-digital-talent/blob/887625996abefe0a857d3b12872975ef8730f244/apps/web/src/pages/Teams/ViewTeamPage/components/ViewTeam.tsx#L39

**Comment by esizer** [comment]
> and the view team page too

Good catch, updated in [update view team page labels](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5921/commits/8f85f8f5bde2ea8d3b82263b36c4bd62a3da20a5)

File: apps/web/src/lang/fr.json [hunk 1]
```diff
@@ -6806,12 +6782,28 @@
...
+  "aGBFm5": {
+    "defaultMessage": "Brève description de l’équipe (Anglais)",
+    "description": "Label for the English team description input"
+  },
+  "n4pt+y": {
+    "defaultMessage": "Nom de l’équipe (Anglais))",
+    "description": "Label for the English team display name input"
+  },
+  "rdYjIb": {
+    "defaultMessage": "Nom de l’équipe (Anglais))",
```
Comment (vd1992) [inline apps/web/src/lang/fr.json:6806]:
```suggestion
    "defaultMessage": "Nom de l’équipe (Anglais)",
```

File: apps/web/src/lang/fr.json [hunk 2]
```diff
@@ -6806,12 +6782,28 @@
...
   },
-  "mOS8rj": {
-    "defaultMessage": "Équipe principale",
-    "description": "Label displayed for selecting what team a new pool belongs to."
+  "aGBFm5": {
+    "defaultMessage": "Brève description de l’équipe (Anglais)",
+    "description": "Label for the English team description input"
+  },
+  "n4pt+y": {
+    "defaultMessage": "Nom de l’équipe (Anglais))",
```
Comment (vd1992) [inline apps/web/src/lang/fr.json:6802]:
```suggestion
    "defaultMessage": "Nom de l’équipe (Anglais)",
```

Comment (vd1992) [reply apps/web/src/lang/fr.json:6802]: missed that there was a second instance of that double ), whoops

Comment (vd1992) [reply apps/web/src/lang/fr.json]: 🙃

File: apps/web/src/lang/fr.json [hunk 3]
```diff
@@ -6806,12 +6782,28 @@
     "defaultMessage": "Nom du gestionnaire",
     "description": "Label for the user select field on team membership form"
   },
-  "hr/i9h": {
-    "defaultMessage": "Sélectionnez une équipe...",
-    "description": "Placeholder displayed for team selection input."
```
Comment (vd1992) [inline apps/web/src/lang/fr.json:6811]: this is new please add it back in

File: apps/web/src/lang/fr.json [hunk 4]
```diff
@@ -6806,12 +6782,28 @@
...
+    "defaultMessage": "Brève description de l’équipe (Français)",
+    "description": "Label for the French team description input"
+  },
+  "YjMfWZ": {
+    "defaultMessage": "Nom de l’équipe (Français)",
+    "description": "Name of an organization/team in French field."
   },
-  "mOS8rj": {
-    "defaultMessage": "Équipe principale",
-    "description": "Label displayed for selecting what team a new pool belongs to."
```
Comment (vd1992) [inline apps/web/src/lang/fr.json:6815]: this too

**Review by vd1992 — CHANGES_REQUESTED** [review]
think I see the reason for the translation check failure, the translations below I added in just recently so definitely should not be deleted

File: apps/web/src/lang/fr.json (hunk 3 above)
Comment (esizer) [reply apps/web/src/lang/fr.json]: dang, I'm going to blame turbo cache. Added back in [add lost translations back](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5921/commits/657b8652fd1d68848c65a3f5c5d5a879b4a536f4)

File: apps/web/src/lang/fr.json (hunk 4 above)
Comment (esizer) [reply apps/web/src/lang/fr.json]: Added back in [add lost translations back](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5921/commits/657b8652fd1d68848c65a3f5c5d5a879b4a536f4)

**Review by vd1992 — APPROVED** [review]
that got a little messy, but I think it's all good now

**Pull request merged**